core/modules/taxonomy/taxonomy.module

```diff
@@ -1110,6 +1110,8 @@ function taxonomy_autoload_info() {
     'TaxonomyTerm' => 'taxonomy.entity.inc',
     'TaxonomyTermController' => 'taxonomy.entity.inc',
     'TaxonomyVocabulary' => 'taxonomy_vocabulary.class.inc',
+    // Layout plugins.
+    'TaxonomyTermDepthLayoutAccess' => 'plugins/access/taxonomy_term_depth_access.inc',
```

Comment:
Could you move this file to be located at `layout/taxonomy_term_depth_access.inc`? Similar to Views, the more complex directory structure is only used within Layout module's own directory where there are lots of handlers.

Reply:
Done